tests/models/wav2vec2_with_lm/test_processor_wav2vec2_with_lm.py

```diff
@@ -157,6 +157,27 @@ def test_feature_extractor(self):
         for key in input_feat_extract.keys():
             self.assertAlmostEqual(input_feat_extract[key].sum(), input_processor[key].sum(), delta=1e-2)
 
+    def test_another_feature_extractor(self):
```

Comment:
We should also test that a feature extractor this isn't SeamlessMT4 or Wav2vec2 raises an error on instantiation